src/main/java/org/gridsuite/casevalidation/server/LoadFlowCaseValidationService.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.casevalidation.server;
+
+import com.powsybl.iidm.network.Network;
+import com.powsybl.loadflow.LoadFlow;
+import com.powsybl.loadflow.LoadFlowParameters;
+import com.powsybl.loadflow.LoadFlowResult;
+import com.powsybl.network.store.client.NetworkStoreService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+@ComponentScan(basePackageClasses = {NetworkStoreService.class})
+@Service
+class LoadFlowCaseValidationService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoadFlowCaseValidationService.class);
+
+    LoadFlowCaseValidationReport validate(Network network, LoadFlowParameters params) {
+        LoadFlowResult result = LoadFlow.run(network, params);
```

Comment:
I think we should rely on loadflow server service and not directly use the LoadFlow java API, so that when we want to update the loadflow we ony have to do it in one micro service. I don't known why we did it like this...

Reply:
Done